content/components/selectpanel.mdx

```diff
@@ -8,20 +8,12 @@ description: Select panel is a semantic dialog that allows for complex selection
 import {Box} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
-import {AccessibilityLink} from '~/src/components/accessibility-link'
-
-<Note>
-  This component is currently under development in React and Rails and might not have all its functionality yet.
-  Internally this epic can be tracked <a href="https://github.com/github/primer/issues/2396">here</a>.
-</Note>
 
 ## Overview
 
-The select panel allows you to select one or multiple items from a list, and it can be enhanced with additional controls like a search input to expedite the selection process. It is often used to augment information of a wide variety of items.
+The select panel enables users to choose one or multiple items from a list and can be enhanced with additional controls like a search input to speed up selection. Offering more flexibility than an [Action Menu](/components/action-menu), it stays open after a selection and allows for the inclusion of extra form controls.
```

Comment:
I don't think this conveys the original prose. ActionMenu is different from SelectPanel, the two should not be conflated as they offer different use cases. I don't think it is sufficient to say SelectPanel offers more controls because it implies that ActionMenu could be used in its place, which is, IMO, not true. For example ActionMenu is immediately reactive (and so useful for non-destructive writes, e.g. filtering) while SelectPanel requires the user to make choices then commit.

Reply:
> It is often used to augment information of a wide variety of items.

The initial phrasing is vague and the idea is to be more specific, but if it's not correct, that obviously doesn't help. Is there a good, distinct way of explaining the difference between the two to avoid any ambiguity here that you could think of?